### PR TITLE
Fix: enforce threshold being set

### LINKED ISSUE #2511: Multisig. Can create without threshold.
## Description

User can create multisig wallet without threshold

Example:

https://github.com/user-attachments/assets/dda9e95d-f557-43aa-8fc7-6eebeac31894

## Steps to Reproduce

1. Go to Multisig creation flow
2. Add creator account
3. Add signatories
4. Press Continue button

## Expected Behavior
Can't proceed with empty threshold, show error

## Actual Behavior
Can proceed

## Environment
https://github.com/novasamatech/nova-spektr/commit/0a3e32d5edae91d623fe09e5f930055e615e403a

### PR DESCRIPTION
**Pull request by Tbaut**
closes #2511

### CODE REVIEW

File: src/renderer/shared/i18n/locales/en.json [hunk 1]
```diff
@@ -187,6 +187,8 @@
     },
     "successMessage": "Wallet added",
     "thresholdDescription": "Set the threshold of signatories required to execute a transaction",
+    "thresholdErrorTitle": "Invalid threshold",
+    "thresholdErrorDescription": "Threshold must be set and >= 2",
```
Comment (Asmadek) [inline src/renderer/shared/i18n/locales/en.json]: Is it confirmed translation? Looks like `>=` can be not clear for users.

File: src/renderer/widgets/CreateWallet/ui/MultisigWallet/SelectSignatoriesThreshold.tsx [hunk 2]
```diff
@@ -46,7 +46,9 @@ export const SelectSignatoriesThreshold = () => {
   const hasOwnedSignatory = !!ownedSignatoriesWallets && ownedSignatoriesWallets?.length > 0;
   const hasEnoughSignatories = signatories.length >= 2;
   const hasEmptySignatory = signatories.map(({ address }) => address).includes('');
-  const canSubmit = hasOwnedSignatory && hasEnoughSignatories && !multisigAlreadyExists && !hasEmptySignatory;
+  const isThresholdValid = threshold.value >= 2 && threshold.value <= signatories.length;
```
Comment (Asmadek) [inline src/renderer/widgets/CreateWallet/ui/MultisigWallet/SelectSignatoriesThreshold.tsx]: Maybe add constant for 2 as `MIN_THRESHOLD` and use it in other places? In line 47 for example

**Review by Asmadek — APPROVED** [review]

File: src/renderer/shared/i18n/locales/en.json (hunk 1 above)
Comment (Tbaut) [reply src/renderer/shared/i18n/locales/en.json]: It's not confirmed, I can change for "superior than 2"

File: src/renderer/widgets/CreateWallet/ui/MultisigWallet/SelectSignatoriesThreshold.tsx (hunk 2 above)
Comment (Tbaut) [reply src/renderer/widgets/CreateWallet/ui/MultisigWallet/SelectSignatoriesThreshold.tsx]: Good point. The 2 is set in stone in the pallets, so we can use it in several places. I'll change this.

**Review by johnthecat — APPROVED** [review]

**Pull request merged**